js/app/app.js

```diff
@@ -134,11 +129,16 @@ var appGFW = {
-    e.$slides.removeClass("slick-slide slick-active slick-visible").attr("aria-hidden","true").css("width","")},Slick.prototype.unslick=function(){var i=this;i.destroy()},Slick.prototype.updateArrows=function(){var i,e=this;i=Math.floor(e.options.slidesToShow/2),e.options.arrows===!0&&e.options.infinite!==!0&&e.slideCount>e.options.slidesToShow&&(e.$prevArrow.removeClass("slick-disabled"),e.$nextArrow.removeClass("slick-disabled"),0===e.currentSlide?(e.$prevArrow.addClass("slick-disabled"),e.$nextArrow.removeClass("slick-disabled")):e.currentSlide>=e.slideCount-e.options.slidesToShow&&e.options.centerMode===!1?(e.$nextArrow.addClass("slick-disabled"),e.$prevArrow.removeClass("slick-disabled")):e.currentSlide>=e.slideCount-1&&e.options.centerMode===!0&&(e.$nextArrow.addClass("slick-disabled"),e.$prevArrow.removeClass("slick-disabled")))},Slick.prototype.updateDots=function(){var i=this;null!==i.$dots&&(i.$dots.find("li").removeClass("slick-active").attr("aria-hidden","true"),i.$dots.find("li").eq(Math.floor(i.currentSlide/i.options.slidesToScroll)).addClass("slick-active").attr("aria-hidden","false"))},Slick.prototype.visibility=function(){var i=this;document[i.hidden]?(i.paused=!0,i.autoPlayClear()):(i.paused=!1,i.autoPlay())},i.fn.slick=function(){var i,e=this,t=arguments[0],o=Array.prototype.slice.call(arguments,1),s=e.length,n=0;for(n;s>n;n++)if("object"==typeof t||"undefined"==typeof t?e[n].slick=new Slick(e[n],t):i=e[n].slick[t].apply(e[n].slick,o),"undefined"!=typeof i)return i;return e},$footer.slick({infinite:!0,slidesToShow:5,slidesToScroll:5,speed:500,autoplay:!0,autoplaySpeed:3e3,slide:"li"})})});    
+    setInterval(slider.next.bind(slider), 3000);
```

Comment:
When you use the arrows we need to clear the interval to prevent unnecessary movements